src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure implements FindArea {
+    private int firsLeg;
+    private int secondLeg;
+    private double thirdLeg;
+
+    public RightTriangle() {
+        super();
+        this.firsLeg = new Random().nextInt(1, 10);
+        this.secondLeg = new Random().nextInt(1, 10);
+        this.thirdLeg = Math.sqrt(Math.pow(firsLeg, 2) + Math.pow(secondLeg, 2));
+    }
+
+    @Override
+    public double getArea() {
+        return (double) firsLeg * secondLeg / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("\nRight triangle "
+                + "color: " + this.getColor()
+                + " area: " + this.getArea()
+                + " first leg: " + firsLeg
```

Comment:
The variable name `firsLeg` is misspelled. It should be `firstLeg` for clarity and consistency.